cloud_function/deploy.sh

```diff
@@ -6,10 +6,10 @@ TRIGGER_BUCKET=${2}
 TRIGGER_EVENT=${3:-"google.storage.object.finalize"}
 REGION=${4:-"us-central1"}
 
-if [ "${GCLOUD_PROJECT}" == "broad-gotc-dev" ]; then
+if [ "${GCLOUD_PROJECT}" == "broad-gotc-dev-storage" ]; then
   # This service account must have access to the WFL API
   SA_EMAIL="aou-cloud-fn-non-prod@broad-gotc-dev.iam.gserviceaccount.com"
```

Comment:
This service account should be "aou-cloud-fn-non-prod@broad-gotc-dev-storage.iam.gserviceaccount.com" now!

Reply:
Omg we both made the same comment at the same time

Reply:
😆